### PR TITLE
MGMT-16075: Add presubmit job for OCP 4.15 in assisted-test-infra

### PR DESCRIPTION
**Pull request by danmanor**
Add `presubmit` job for `OCP 4.15` in assisted-test-infra

### CODE REVIEW

File: ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-master.yaml [hunk 1]
```diff
@@ -233,6 +233,14 @@ tests:
         OPENSHIFT_VERSION=4.14
       TEST_SUITE: full
     workflow: assisted-baremetal
+- as: e2e-metal-assisted-4-15
+  optional: true
```
Comment (adriengentil) [inline ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-master.yaml:237]:
```suggestion
  optional: true
  always_run: false
```

Comment (danmanor) [reply ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-master.yaml:237]: I don't see any `always_run: false` in the rest of the tests but they don't run automatically no?

Comment (adriengentil) [reply ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-master.yaml:237]: > before we were updating the job directly
here: https://github.com/openshift/release/pull/45375/files#diff-e52081e3ba3e344ffd4d8ffb587d28fcb4366b5207c692138f512a11b8f7d404R484

Comment (danmanor) [reply ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-master.yaml:237]: Oh because it is new. got it

**Comment by adriengentil** [comment]
you need you run a `make update` I guess

**Review by adriengentil — APPROVED** [review]

**Review by adriengentil — APPROVED** [review]

**Comment by danmanor** [comment]
/pj-rehearse pull-ci-openshift-assisted-test-infra-master-e2e-metal-assisted-4-15

**Comment by danmanor** [comment]
/pj-rehearse pull-ci-openshift-assisted-test-infra-master-e2e-metal-assisted-4-15

**Pull request merged**